donate.html

```diff
@@ -90,9 +90,15 @@ <h2>Supporters</h2>
                         <a href="https://unraid.net/" target="_blank"><img src="img/unraid-logo-only.png" title="Unraid; Generous monetary donations, and provided us with some license keys for their software" alt="Unraid Logo" class="supporter-logo" /></a>
                     </p>
                 </div>
+                <div class="col-md-12 text-center">
```

Comment:
```suggestion
                <div class="col-md-12 text-center mt-5">
```
Adding a bit of margin to the top should help separate this header from the images above.